packaging/nnstreamer.spec

```diff
@@ -472,6 +472,25 @@ Requires:	nnstreamer = %{version}-%{release}
 Requires:	grpc-devel
 %description  grpc
 NNStreamer's tensor_source/sink plugins for gRPC support.
+
```

Comment:
Wait a sec... do we need grpc-devel for grpc? (LIne 473)

Fot Tizen, please add ```Recommends:``` on grpc-protobuf and grpc-flatbuf from grpc.

Reference: https://fedoraproject.org/wiki/Packaging:WeakDependencies

For fine control, you need to understand the following dependency keywords:
Requires, Recommends, Suggests,
Provides, Supplements, Enhances.

Note that TIzen heavily depends on the usages of Requires, Recommends, Suggests, and Provides.

Reply:
Oh.. my bad. don't need grpc-devel